lib/rouge/lexers/dafny.rb

```diff
@@ -15,12 +15,12 @@ class Dafny < RegexLexer
         break calc case class codatatype const constructor
         datatype decreases default
```

Comment:
Missing keywords:
* `allocated`
* `by`
* `export`
* `reveal` (in addition to `reveals`, which is already mentioned)
* `witness`

I would suggest also listing the following as keywords, even though--technically--they are only contextual keywords:
* `to`
* `downto`

These are no longer keywords:
* `default`
* `inductive`
* `where`

Missing type:
* `ORDINAL`

Reply:
done